packages/gatsby-theme-carbon/src/components/Caption/caption.scss

```diff
@@ -11,8 +11,8 @@ img + .#{$prefix}--caption {
 }
 
 // Caption after video
-[class^='Video-module--video'] + .#{$prefix}--caption {
-  margin-top: -$spacing-08;
+div[class^='Video-module--video'] + .#{$prefix}--caption {
+  margin-top: -4rem;
```

Comment:
just wondering, we don't have a token for this?

Reply:
Good call, I can change to `$layout-05` we just don't have a spacing token that goes up to 4rem

Reply:
or maybe the mini-units mixin?